pkgs/development/interpreters/php/5.4.nix

```diff
@@ -9,7 +9,7 @@ in
 
 composableDerivation.composableDerivation {} ( fixed : let inherit (fixed.fixed) version; in {
 
-  version = "5.4.35";
+  version = "5.6.5";
```

Comment:
The file is called 5.4. And you should not bump php like that in the same derivation, because it's not supposed to be backward compatible.